### ISSUE TITLE
How to edit what the page will look like below the url bar?

### ISSUE THREAD

**Issue by Benji-Collins** [Athlon1600/php-proxy-app#94]
Is there a way to change what the proxified page will look like after it has loaded? For example if I wanted to add an element to the start of end of it?

**Comment by zpit367** [comment]
That would be cool, maybe even a plugin to add things to header/footer?

**Comment by ash121121** [comment]
you could use my popup plugin and just put anything in the html file. or use my proxystripper plugin and just add stuff in that way

**Comment by zpit367** [comment]
Cool thanks!

**Comment by Benji-Collins** [comment]
@ash121121 whenever I try to load a page with the AdsPlugin added to the config.php file I get a HTTP 500 error, server can not handle the request. I have added it in the same format as all other plugins there: `'AdsPlugin'`. Then I add a comma to the end of the twitter one: `'Twitter',` so the array still works. 

The path to the ads.html file is something I am not totally sure on but I have tried just `/templates/ads.html`, `/path/all/the/way/from/root/Sneakytest/templates/ads.html` and `./templates/ads.html`. Am I getting that wrong which is causing the whole proxy to stop working?

While testing I have just put a <p>Stuff</p> in the ads.html file.
